docs/monitoring/flow.rst

```diff
@@ -61,8 +61,8 @@ Another useful notification rule plugin that can be used when processing statist
 | |MonitoredBuffered| |
 +---------------------+
 
-In the above example it is shown monitoring the BUFFERED statistic, the is the number of readings written to the storage layer and represents the aggregate of all the data read by all of the south services. This can then be used to monitored if the ingest for the Fledge in total moves outside of the observed operating criteria that has been measured previously. 
+In the above example it is shown monitoring the BUFFERED statistic. This is the number of readings written to the storage layer and represents the aggregate of all the data read by all of the south services. This can then be used to monitor if the ingest for the Fledge in total moves outside of the observed operating criteria that has been measured previously. 
 
 .. note::
 
-   This is very similar to the automated monitoring in each south service, although because it is unaware of configured changes to the collection rate it has the disadvantage that it is unable to distinguish from a configured change to a change the occurs because of some failure of equipment or of Fledge.
+   This is very similar to the automated monitoring in each south service. Because it is unaware of configured changes to the collection rate it has the disadvantage that it is unable to distinguish from a configured change to a change the occurs because of some failure of equipment or of Fledge.
```

Comment:
"...to a change the occurs because of..." -> "...to a change that occurs because of..."